tiatoolbox/annotation/storage.py

```diff
@@ -2383,10 +2469,10 @@ def query(
 
     def bquery(
         self,
-        geometry: Optional[QueryGeometry] = None,
-        where: Union[str, bytes, Callable[[Geometry, Dict[str, Any]], bool]] = None,
-        min_area: Optional[float] = None,
-    ) -> Dict[str, Tuple[float, float, float, float]]:
+        geometry: QueryGeometry | None = None,
+        where: str | bytes | Callable[[Geometry, dict[str, Any]], bool] | None = None,
```

Comment:
we have some custom types we can use here, I think we could replace this with Predicate | None

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/625/commits/991322d859c0c22e7c7a1d713362944aa4317198